modules/complex-number/test/test_Nikolai_complex_number.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Sokolov Nikolai
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Nikolai_Sokolov_ComplexNumberTest, Can_Create_Zero) {
+    // Arrange
+    double re = 0.0;
+    double im = 0.0;
+
+    // Act
+    ComplexNumber z(re, im);
+
+    // Assert
+    EXPECT_EQ(re, z.getRe());
+    EXPECT_EQ(im, z.getIm());
+}
+
+TEST(Nikolai_Sokolov_ComplexNumberTest, Plass_Operator) {
```

Comment:
Было бы хорошо переименовать тест
```suggestion
TEST(Nikolai_Sokolov_ComplexNumberTest, Add_Operator) {
```